### PR TITLE
Improved: UI for lat-long card, UI for facility management, UI for Parking and Groups, button text

### PR DESCRIPTION
**Pull request by amansinghbais**
### Related Issues

Closes #

### Short Description and Why It's Useful

Improved:
- UI of latitude and longitude card in facility details page.
- UI of Parking page.
- UI of FindGroups page.
- button text message of segments.

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2023-12-12 11-06-29
Screenshot from 2023-12-12 11-07-26 Screenshot from 2023-12-12 11-07-31

Screenshot from 2023-12-12 11-07-57

Screenshot from 2023-12-12 11-08-32

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/facilities#contribution-guideline)

### CODE REVIEW

File: src/views/FacilityDetails.vue [hunk 1]
```diff
@@ -1113,9 +1115,7 @@ ion-card-header {
 }
 
 
-/*Height of segment is defined now since their are less list items. Will remove it later */
 .segments {
```
Comment (dt2patel) [inline src/views/FacilityDetails.vue:1118]: do we need to use a class or can we just select the element? @amansinghbais

File: src/views/FindGroups.vue [hunk 2]
```diff
@@ -12,34 +12,32 @@
...
-                {{ translate('Create group') }}
+        <main class="groups" v-if="groups.length">
+          <!-- custom sorting in the UI to keep OMS_FULFILLMENT and PICKUP types first -->
+          <ion-card v-for="(group, index) in customSort(groups, ['OMS_FULFILLMENT', 'PICKUP'], 'facilityGroupId')" :key="index">
+            <ion-item lines="full">
+              <ion-label>
+                <h1>{{ group.facilityGroupName }}</h1>
+                <p>{{ group.facilityGroupId }}</p>
+              </ion-label>
+              <ion-button @click="openFacilityGroupActionsPopover($event, group)" fill="clear" color="medium">
```
Comment (dt2patel) [inline src/views/FindGroups.vue:23]: need to add slot here

File: src/views/FindGroups.vue [hunk 3]
```diff
@@ -12,34 +12,32 @@
...
-              </ion-item>
-              <ion-item lines="full" v-if="group.description">
-                <ion-label>{{ group.description }}</ion-label>
-              </ion-item>
-            </ion-card> 
-            <ion-card>
-              <ion-button color="medium" fill="clear" @click="openCreateFacilityGroupModal()">
-                <ion-icon :icon="addOutline" slot="start"/>
-                {{ translate('Create group') }}
+        <main class="groups" v-if="groups.length">
```
Comment (dt2patel) [inline src/views/FindGroups.vue:15]: instead of using a class why not select the main element when it has cards?

**Review by dt2patel — CHANGES_REQUESTED** [review]
Looks good, minor suggestions

File: src/views/FacilityDetails.vue (hunk 1 above)
Comment (amansinghbais) [reply src/views/FacilityDetails.vue:1118]: @dt2patel sir,  div with segments class contains 3 child div inside it and some css is applied to those div tag with the help of 'segments' class as ".segments div". So if we don' t want to use segments class, we need to give some identifier(class) to the div inside for adding css on them.

File: src/views/FindGroups.vue (hunk 2 above)
Comment (amansinghbais) [reply src/views/FindGroups.vue]: Resolved

File: src/views/FindGroups.vue (hunk 3 above)
Comment (dt2patel) [reply src/views/FindGroups.vue:15]: Yes it can be generalized in variables file

**Review by dt2patel — APPROVED** [review]
Looks good

**Pull request merged**